src/index.html

```diff
@@ -7,8 +7,26 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a href="#home" class="logo"><img src="./images/logo.png" alt="MOYO"></a>
+      <nav class="navbar">
+        <ul class="nav__list">
+          <li><a class="nav__link is-active first" href="#apple">Apple</a></li>
+          <li><a class="nav__link" href="#samsung">Samsung</a></li>
```

Comment:
please fix code formatting, e.g.:

```suggestion
          <li>
            <a class="nav__link" href="#samsung">Samsung</a>
          </li>
          <li>
            <a class="nav__link" href="#smartphones">Smartphones</a>
          </li>
          <li>
            <a class="nav__link" data-qa="hover">Laptops &amp; computers</a>
          </li>
          <li>
            <a class="nav__link" href="#gadgets">Gadgets</a>
          </li>
```

Reply:
Not resolved. Fix formatting. Check all other similar cases.